vocabularies/Common.xml

```diff
@@ -155,6 +155,14 @@
         <Annotation Term="Core.RequiresType" String="Edm.String" />
       </Term>
 
+      <Term Name="IsISOCurrency" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property Parameter">

```

Comment:
Given the lengthy discussions around "internal" currency codes, do we want to have the infix `ISO` here?

Reply:
Done